boskos/boskos.go

```diff
@@ -155,7 +155,7 @@ func handleAcquire(r *ranch.Ranch) http.HandlerFunc {
 		resource, err := r.Acquire(rtype, state, dest, owner)
 
 		if err != nil {
-			logrus.WithError(err).Errorf("No available resource")
+			logrus.WithError(err).Warningf("No available resource")
```

Comment:
I think you can check err type here (https://github.com/kubernetes/test-infra/blob/master/boskos/ranch/ranch.go#L46-L82) and warn if it's a not found error

Reply:
done